pkg/config/lima_config_applier.go

```diff
@@ -154,7 +154,18 @@ func toggleUserModeEmulationInstallationScript(limaCfg *limayaml.LimaYAML, enabl
 			Mode: "system",
 			Script: fmt.Sprintf(`%s
 #!/bin/bash
-dnf install -y --setopt=install_weak_deps=False qemu-user-static-aarch64 qemu-user-static-arm qemu-user-static-x86
+qemu_pkgs=""
```

Comment:
we have repeat this section multiple times in other files. Is it possible to create as bash function. something like
https://unix.stackexchange.com/questions/599604/create-reusable-functions-bash
Btw, i am not sure whether this is even doable in your case.